src/main/java/net/library/service/UserService.java

```diff
@@ -1,39 +1,43 @@
 package net.library.service;
 
-import net.library.object.User;
+import net.library.model.entity.User;
+import net.library.repository.UserRepository;
+import net.library.model.dto.request.UsersTest;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.UUID;
+import java.util.stream.StreamSupport;
 
 @Service
 public class UserService {
     private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);
 
-    private final List<User> users = new ArrayList<>();
+    private final List<UsersTest> users = new ArrayList<>();
 
-    public void initUserLst() {
-        users.add(new User("Smith1", "John1", 12));
-        users.add(new User("Smith2", "John2", 22));
-        users.add(new User("Smith3", "John3", 32));
-    }
+    private final UserRepository userRepository;
 
-    public List<User> getUsers() {
-        LOGGER.debug("List of books returned: {} ",users);
-        return users;
+    @Autowired
+    public UserService(UserRepository userRepository){
+        this.userRepository =userRepository;
     }
 
-    public void deleteAllUsers(){
+    public List<User> getAllUsers(){
 
-        users.clear();
-        LOGGER.info("All the users deleted");
+       return   StreamSupport.stream(userRepository.findAll().spliterator(),false).toList();
     }
 
-    public void addUser(User user){
+    public void saveUser(User user){
+
+        userRepository.save(user);
+        LOGGER.info("User saved");
```

Comment:
don't do it. if you need this info you should enable basic logging

Reply:
I do not understand. When you say basic logging what do you mean?